### PR TITLE
English translation

### PR DESCRIPTION
**Pull request by johnmeunier**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,43 +1,43 @@
...
+# Contributing to the everythingToEverywhere library AXA France
 
-Merci de considérer la contribution au repo everythingToEverywhere ! Nous sommes ravis de voir que vous souhaitez participer à l'amélioration de ce projet.
+Thank you for considering contributing to the everythingToEverywhere repo! We are excited to see that you want to participate in improving this project.
 
-## Démarrage du projet
+## Getting Started
 
-Le build du projet se fait via la commande `npm run build`. Pour tester le package en local, vous pouvez faire une installation locale grâce à la commande `npm i <local_path>`.
+The project build is done via the command `npm run build`. To test the package locally, you can perform a local installation using the command `npm i <local_path>`.
```
Comment (Julien-Gaudet-Axa) [inline CONTRIBUTING.md:7]: Suggestion: "The project build is done **using** the command"

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -1,43 +1,43 @@
...
-3. **Créer une Branche :** Créez une branche pour travailler sur votre nouvelle fonctionnalité ou correction.
+3. **Create a Branch:** Create a branch to work on your new feature or fix.
 
 ```bash
-git switch -c nom-de-ma-branche
+git switch -c my-branch-name
 ```
 
-1. **Faire des Modifications :** Effectuez les modifications nécessaires et assurez-vous que votre code respecte les conventions du projet. Utilisez la [Conventional Commits](https://www.conventionalcommits.org/) pour le nommage et la découpe des commits.
+4. **Make Changes:** Make the necessary changes and ensure your code adheres to the project's conventions. Use [Conventional Commits](https://www.conventionalcommits.org/) for naming and splitting commits.
```
Comment (Julien-Gaudet-Axa) [inline CONTRIBUTING.md:25]: Suggestion: "and ensure your code **follows the** project's conventions."

File: README.md [hunk 3]
```diff
@@ -95,16 +91,16 @@ defineCustomElementReact({
 })
 ```
 
-### Utiliser le nouveau custom element
+### Using the new custom element
 
-Dans notre premier exemple, le composant React s'appelait `AnyReactComponent` (en pascal case).
-Le custom element doit répondre aux normes de l'HTML et aux bonnes pratiques en vigueur, donc avec un namespace et en pascal case.
+In our previous example, the React component was called `AnyReactComponent` (in Pascal case). 
+The custom element must comply with HTML standards and best practices, so it should have a namespace and be in Pascal case.
```
Comment (Julien-Gaudet-Axa) [inline README.md:97]: You mean kebab case no ?  

kebab-case: owc-any-react-component

PascalCase: OwcAnyReactComponent

Comment (johnmeunier) [reply README.md]: thanks ! done

File: CONTRIBUTING.md (hunk 2 above)
Comment (johnmeunier) [reply CONTRIBUTING.md]: thanks ! done

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -1,43 +1,43 @@
-# Contribuer à la librairie everythingToEverywhere AXA France
+# Contributing to the everythingToEverywhere library AXA France
 
-Merci de considérer la contribution au repo everythingToEverywhere ! Nous sommes ravis de voir que vous souhaitez participer à l'amélioration de ce projet.
+Thank you for considering contributing to the everythingToEverywhere repo! We are excited to see that you want to participate in improving this project.
```
[the remark this answers is not in the document]
Comment (johnmeunier) [reply CONTRIBUTING.md]: thanks ! done

File: CODE_OF_CONDUCT.md [hunk 5]
```diff
@@ -1,46 +1,46 @@
...
+Project maintainers have the right and responsibility to remove, edit, or reject comments, contributions, code, wiki edits, issues, and other contributions that do not adhere to this Code of Conduct, or to temporarily or permanently ban any contributor for behaviors that do not comply with this Code of Conduct.
 
-## Portée
+## Scope
 
-Ce Code de Conduite s'applique à la fois au sein des espaces du projet ainsi que dans les espaces publics lorsqu'un individu représente le projet ou sa communauté. Font partie des exemples de représentation d'un projet ou d'une communauté le fait d'utiliser une adresse e-mail du projet, de poster via un compte officiel sur les réseaux sociaux, ou d'intervenir en tant que représentant désigné lors d'un événement en ligne ou hors ligne.
+This Code of Conduct applies both within project spaces and in public spaces when an individual is representing the project or its community. Examples of representing a project or community include using a project email address, posting via an official social media account, or acting as a designated representative at an online or offline event.
```
[the remark this answers is not in the document]
Comment (johnmeunier) [reply CODE_OF_CONDUCT.md]: thanks ! done

File: CODE_OF_CONDUCT.md [hunk 6]
```diff
@@ -1,46 +1,46 @@
...
 
-## Nos responsabilités
+## Our Responsibilities
 
-Les mainteneurs du projet ont la responsabilité de clarifier les normes de comportement acceptable et doivent prendre des mesures correctives appropriées et équitables en réponse à tout cas de comportement inacceptable.
+Project maintainers are responsible for clarifying the acceptable standards of behavior and must take appropriate and fair corrective action in response to any instances of unacceptable behavior.
 
-Les mainteneurs du projet ont le droit et la responsabilité de supprimer, éditer ou rejeter les commentaires, les contributions, le code, les modifications du wiki, les problèmes et autres contributions qui ne respectent pas ce Code de Conduite, ou de bannir temporairement ou définitivement tout contributeur pour des comportements autres que ceux respectant ce Code de Conduite.
+Project maintainers have the right and responsibility to remove, edit, or reject comments, contributions, code, wiki edits, issues, and other contributions that do not adhere to this Code of Conduct, or to temporarily or permanently ban any contributor for behaviors that do not comply with this Code of Conduct.
```
[the remark this answers is not in the document]
Comment (johnmeunier) [reply CODE_OF_CONDUCT.md]: thanks ! done

File: CODE_OF_CONDUCT.md [hunk 7]
```diff
@@ -1,46 +1,46 @@
-# Code de Conduite
+# Code of Conduct
 
-## Notre engagement
+## Our Commitment
 
-En tant que participants et contributeurs, nous nous engageons à rendre la participation à notre projet et à notre communauté une expérience sans harcèlement pour tout le monde, indépendamment de l'âge, de la taille, de l'invalidité, de l'ethnicité, de l'identité de genre et de l'expression, du niveau d'expérience, de l'éducation, du statut socio-économique, de la nationalité, de l'apparence personnelle, de la race, de la religion ou de l'orientation sexuelle.
+As participants and contributors, we pledge to make participation in our project and community a harassment-free experience for everyone, regardless of age, size, disability, ethnicity, gender identity and expression, level of experience, education, socio-economic status, nationality, personal appearance, race, religion, or sexual orientation.
```
[the remark this answers is not in the document]
Comment (johnmeunier) [reply CODE_OF_CONDUCT.md]: thanks ! done

**Pull request merged**